lib/Dialect/EmitC/Conversion/TosaToEmitC.cpp

```diff
@@ -122,6 +282,14 @@ void populateTosaToEmitcPatterns(MLIRContext *ctx,
 
   // Binary elementwise ops
   patterns.insert<CallOpConversion<tosa::AddOp>>(ctx, "tosa::add");
+  patterns.insert<CallOpConversion<tosa::MulOp>>(ctx, "tosa::mul");
+
+  patterns.insert<CallOpBroadcastableConversion<tosa::AddOp>>(ctx, "tosa::add");
+  patterns.insert<MulOpConversion>(ctx, "tosa::mul");
```

Comment:
Adding `patterns.insert<CallOpConversion<tosa::MulOp>>(ctx, "tosa::mul");` *and* `patterns.insert<MulOpConversion>(ctx, "tosa::mul");` seems wrong.